core/src/main/java/com/afollestad/materialdialogs/internal/main/DialogTitleLayout.kt

```diff
@@ -114,6 +115,7 @@ class DialogTitleLayout(
     val titleHalfHeight = titleView.measuredHeight / 2
     val titleTop = contentMidPoint - titleHalfHeight
     val titleBottom = contentMidPoint + titleHalfHeight
+    + titleView.maybeSetAdditionalPadding(titleView.measuredHeight)
```

Comment:
Looks like `+ titleView.maybeSetAdditionalPadding(titleView.measuredHeight)` actually doesn't get used when it's formatted like this, but I'll fix it in a separate commit :) 

Should be

```kotlin
val titleBottom = contentMidPoint + titleHalfHeight +
        titleView.maybeSetAdditionalPadding(titleView.measuredHeight)
```